OpenEdXMobile/src/main/java/org/edx/mobile/util/NotificationUtil.java

```diff
@@ -0,0 +1,53 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.iid.FirebaseInstanceId;
+import com.google.firebase.iid.InstanceIdResult;
+import com.google.firebase.messaging.FirebaseMessaging;
+
+import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.notifications.services.NotificationService;
+
+public class NotificationUtil {
+
+    /**
+     * Subscribe to the Topic channels that will be used to send Group notifications
+     * @param environment
+     */
+    public static void subscribeToTopics(IEdxEnvironment environment, Context context){
+        if(environment.getConfig().areFirebasePushNotificationsEnabled()) {
+            FirebaseMessaging.getInstance().subscribeToTopic(
+                    NotificationService.NOTIFICATION_TOPIC_RELEASE
+            );
+            Toast toast = Toast.makeText(context, "Test", Toast.LENGTH_LONG);
+            toast.show();
+        }
+    }
+
+    public static void logFirebaseToken(){
+        // Get token
+        FirebaseInstanceId.getInstance().getInstanceId()
+            .addOnCompleteListener(new OnCompleteListener<InstanceIdResult>() {
+                @Override
+                public void onComplete(@NonNull Task<InstanceIdResult> task) {
+                Logger logger = new Logger("NotificationUtils");
```

Comment:
This should be declared at the top of the class like you've done in `NotificationService` class.